### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by oleg-gord**
- [DEMO LINK](https://oleg-gord.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://oleg-gord.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+.label::before {
+  content: "";
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  background-image: url("./images/Search.svg");
+  background-size: contain;
+  z-index: 1;
```
Comment (plodnikate) [inline src/style.css:42]: Please, check and follow this approach [css-order](https://9elements.com/css-rule-order/), fix it in all places.

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,34 @@
...
+      <label class="label">
+        <input
+          class="search"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form
```
Comment (plodnikate) [inline src/index.html:29]: Add extra line between elements, but not parent and child.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+.search-small {
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.search:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
```
Comment (plodnikate) [inline src/style.css:74]: Use variables.

**Review by plodnikate — CHANGES_REQUESTED** [review]
Good job. Let's improve some points!

**Comment by oleg-gord** [comment]
я не знаю що не так у мене всі тести проходять

**Comment by oleg-gord** [comment]
,

**Comment by oleg-gord** [comment]
..

**Pull request closed without merging**